### DISCUSSION TITLE
Investigate Microsoft based alternative to Google Colab

### DISCUSSION THREAD

**Discussion by kkmcgg** [NSCC-COGS/Aestheta#104] [Ideas]
We should explore the [Microsoft's Offerings for Cloud based Python Development Environments](https://docs.microsoft.com/en-us/azure/notebooks/quickstart-export-jupyter-notebook-project#use-notebooks-in-github-codespaces)

This is not directly related to our code base necessarily. To date we have been discussing and using [google colab](https://colab.research.google.com/) as a baseline web based python develop environment where we can easily cue up a python environment in the browser where libraries like google tensforflow are readily available. 

as we look to processing larger datasets stored on the NSCC-azure/onedrive server (a Microsoft product) It maybe wise to investigate options for Microsoft based alternatives to google colab. Perhaps these tools would integrate better with our data servers, as well as gitHub (which is own by Microsoft), and Micosoft's VS code (the IDE we've been mostly using it seems)

So! It seems that Microsoft has retired its direct alternative to google colab, called [Azure Notebooks ](https://notebooks.azure.com/).

A specific good option may be **GitHub Codespaces** which is currently under a beta and you must [apply for early access.  ](https://github.com/features/codespaces)

**Comment by arraySurfer81210** [comment]
DataLore by JetBrains is similar to Google Colab. Not a Microsoft product. However good to know as DataLore markets themselves as a data-science specialized platform. 

https://datalore.jetbrains.com/notebook/z145g3Zu8Ko2nprHgPlCdX/9LlplYkoKirdbMj1LQk57y/
